document/js-api/index.bs

```diff
@@ -860,15 +858,15 @@ Immediately after a WebAssembly [=memory.grow=] instruction executes, perform th
 
 <div algorithm=dom-Memory-toResizableBuffer>
     The <dfn method for="Memory">toResizableBuffer()</dfn> method, when invoked, performs the following steps:
+    1. If |memtype| does not have a max,
```

Comment:
We should pull up the `Let |memtype| be ...` and the things it depends on from below.